components/fxa-client/src/oauth.rs

```diff
@@ -110,14 +110,17 @@ impl FirefoxAccount {
     /// the pairing authority.
     /// * `scopes` - Space-separated list of requested scopes by the pairing supplicant.
     /// * `entrypoint` - The entrypoint to be used for data collection
+    /// * `metrics` - Optional parameters for metrics
     pub fn begin_pairing_flow(
         &mut self,
         pairing_url: &str,
         scopes: &[&str],
         entrypoint: &str,
+        metrics: MetricsParams,
```

Comment:
@tarikeshaq These are all optional parameters. What are your thoughts on making this a `Option<MetricsParams>` ? this way everywhere you use it you can just do something like `acct.begin_oauth_flow(&SCOPES, "device_api_example", None)?;` This should make it easier for us whereever we use it to avoid including the `ffi` module.

Reply:
Alrighty done!